src/admin/js/login.js

```diff
@@ -1,9 +1,10 @@
+/* globals ADMIN_API_URL */
 function redirect() {
     window.location = '/admin';
 }
 
 // check for user already logged in
-if (window.sessionStorage.getItem('token')) {
+if (window.localStorage.getItem('token')) {
```

Comment:
Why did you switch to localStorage? The session won't be shared between tabs in a same browser, and it's a poor user experience.